### DISCUSSION TITLE
Jira Connection

### DISCUSSION THREAD

**Discussion by kaiserkiwi** [spatie/flareapp.io-roadmap#49] [Ideas]
It would be awesome if it would be possible to connect Jira so that I can create an issue out of an error in Flare. Bonus points if the issue is visible on the Flare entry so my team can easily see what the status of an error is.

**Comment by AlexVanderbist** [comment]
Hey! We're currently considering this. I'll get back to you once I know more :)

**Reply by treestonemedia** [reply]
@AlexVanderbist this would be a game changer for us, any update on this?

**Comment by ChrisCook1906** [comment]
I wanted to echo the value we'd receive from this integration. We'd love to help test or support.

**Comment by AlexVanderbist** [comment]
It's currently on our internal planning! No ETA yet but it's high up there.

**Comment by freekmurze** [comment]
Our team is working on this. ETA: early October 🥳

**Reply by cargix1** [reply]
Can't wait @freekmurze  this will be huge for us! Thanks so much for listening to our feedback :D

**Comment by rubenvanassche** [comment]
It is out!

https://flareapp.io/blog/introducing-jira-integration-with-flare
